testing/vault/vault.post-deinstall

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Delete vault group and user
+delgroup vault 2> /dev/null
+deluser vault 2> /dev/null
```

Comment:
This is not a good idea. You can’t know if the vault user/group has been created by the package or not; user can use it even for something else.